sdk/python/feast/job.py

```diff
@@ -47,8 +46,7 @@ def __init__(
         """
         self.job_proto = job_proto
         self.serving_stub = serving_stub
-        # TODO: abstract away GCP depedency
-        self.gcs_client = storage.Client(project=None)
+        self.staging_strategy = StagingStrategy()
```

Comment:
Somehow `StagingStrategy` does not fit for me here. If I read this I wouldn't know what this class does. Would it be better if we called it something like `ObjectStoreClient`? or `StorageClient`?

Reply:
I will rename class as `StorageClient`.